bucket/ocenaudio.json

```diff
@@ -5,12 +5,9 @@
     "license": "Unknown",
     "architecture": {
         "64bit": {
-            "url": "https://www.ocenaudio.com/downloads/ocenaudio64_portable.zip?version=v3.14",
-            "hash": "3a8d80a904b37103ec03d0226acfa75addc287201e02def009cd0c7f32b57586"
-        },
-        "32bit": {
-            "url": "https://www.ocenaudio.com/downloads/ocenaudio_portable.zip?version=v3.14",
-            "hash": "3a8d80a904b37103ec03d0226acfa75addc287201e02def009cd0c7f32b57586"
+            "url": "https://www.ocenaudio.com/downloads/index.php/ocenaudio_windows64.zip?version=v3.14",
+            "hash": "1b756d68657a803c61b70724aee0a9c437f027cb41761e90dfe3df318902c1f6",
+            "extract_dir": "ocenaudio"
         }
     },
     "bin": "ocenaudio.exe",
```

Comment:
```suggestion
    "bin": "ocenaudio_cli.exe",
```

Reply:
I don't think scoop uses the CLI binary?

Reply:
I tried it out, it actually has a different interface. Don't know why they didn't put that functionality in the main app. Added both exes to bin